projectDocs/dev/createDevEnvironment.md

```diff
@@ -84,7 +84,7 @@ If you aren't sure, run `git submodule update` after every git pull, merge or ch
 
 #### Run time dependencies
 
-* [eSpeak NG](https://github.com/espeak-ng/espeak-ng), version 1.52.0
+* [eSpeak NG](https://github.com/espeak-ng/espeak-ng), version 1.52.0 commit `e93d3a97ee9237f7e170cc8870f44c14de9032a9`
```

Comment:
We're off a tagged version

```suggestion
* [eSpeak NG](https://github.com/espeak-ng/espeak-ng), commit `e93d3a97ee9237f7e170cc8870f44c14de9032a9`
```

Unrelated, but thoughts about removing versions from this section? Maintenance of this is a pain and people can just check the git commit directly in source control